src/test/regress/expected/distributed_functions_0.out

```diff
@@ -0,0 +1,885 @@
+SET citus.next_shard_id TO 20020000;
```

Comment:
> Postgres tightened up its checks for invalid GUC names hence we started
to get an alternative output for one of our tests. We add an alternative
output since the file is relatively small.

The file is 800 lines :/ 


The diff is pretty small:
```
 diff src/test/regress/expected/distributed_functions_0.out src/test/regress/expected/distributed_functions.out
346a347,348
> ERROR:  invalid configuration parameter name "citus.setting;'"
> DETAIL:  Custom parameter names must be two or more simple identifiers separated by dots.
353a356,357
> ERROR:  invalid configuration parameter name "citus.setting;'"
> DETAIL:  Custom parameter names must be two or more simple identifiers separated by dots.
545c549,550
< ERROR:  invalid type name "val1 macaddr"
---
> ERROR:  syntax error at or near "macaddr"
> CONTEXT:  invalid type name "val1 macaddr"
```

I'd even consider these tests as they do not mean much anyway

Reply:
I removed the tests